regression/contracts/assigns_replace_havoc_dependent_targets_fail/enforce.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+main_enforce.c
+--enforce-contract resize_vec _ --signed-overflow-check --unsigned-overflow-check --pointer-overflow-check
+^VERIFICATION SUCCESSFUL$
+^EXIT=0$
+^SIGNAL=0$
+--
+--
+Verifies the contract being replaced in `replace_resize_vec.desc`.
```

Comment:
Is that the correct file name? I didn't find it.

Reply:
fixed